### PR TITLE
CVE-2019-18409.yml for ruby_parser-legacy

### PR DESCRIPTION
**Pull request by fwininger**
Hi,

Please consider this PR for the CVE-2019-18409.

https://nvd.nist.gov/vuln/detail/CVE-2019-18409.

Thanks

### CODE REVIEW

File: gems/ruby_parser-legacy/CVE-2019-18409.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  privilege escalation because of world-writable files. For example,
+  if the brakeman gem (which has a legacy dependency) 4.5.0 through 4.7.0 is used,
+  a local user can insert malicious code into the
+  ruby_parser-legacy-1.0.0/lib/ruby_parser/legacy/ruby_parser.rb file.
+
+cvss_v2: 4.6
+cvss_v3: 7.8
+
+patched_versions:
+  - "> 1.0.0"
```
Comment (reedloden) [inline gems/ruby_parser-legacy/CVE-2019-18409.yml]: Doesn't look like there's a fixed version yet.

**Comment by reedloden** [comment]
@fwininger Is it only version 1.0.0 that is affected?

File: gems/ruby_parser-legacy/CVE-2019-18409.yml (hunk 1 above)
Comment (fwininger) [reply gems/ruby_parser-legacy/CVE-2019-18409.yml]: Yes, there a are no fix yet. :(

**Comment by fwininger** [comment]
@reedloden : yes only version 1.0.0 is affected because it's the only one published yet.

**Pull request merged**